test/event_listeners.spec.coffee

```diff
@@ -215,7 +215,7 @@ describe 'AWS.EventListeners', ->
       makeRequest(->)
       expect(options.timeout).to.equal(15)
 
-    it 'signs only once in normal case', ->
+    it 'signs only once in normal case', (done) ->
```

Comment:
Hopefully, making all of these async should take care of the flickering failures we've been seeing over the past few weeks